contracts/axelar-gateway/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 #![no_std]
 
 mod auth;
+mod error;
```

Comment:
```suggestion
pub mod error;
```
so you can access errors in integration tests